### PR TITLE
Site edits

### PR DESCRIPTION
**Pull request by tedmcox**
Removed the open CFP info from get_involved.html and added OSEM schedule link to 2015.html.

### CODE REVIEW

File: schedule/2015.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
 <div class="col-md-12">
     <h1>SeaGL 2015 Schedule</h1>
 
-    <p>Official schedule will be published after talks are selected.</p>
-    <p>Visit the <a href="https://osem.seagl.org/">CFP</a> to submit a talk!</p>
+    <p>The official SeaGL schedule is published on OSEM:
+    
+    <https://osem.seagl.org/conference/seagl2015/schedule>
```
Comment (ex-nerd) [inline schedule/2015.html]: Looks like you missed the `a href` part of this.

**Comment by tedmcox** [comment]
Fixed. Thanks!

File: schedule/2015.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
 <div class="col-md-12">
     <h1>SeaGL 2015 Schedule</h1>
 
-    <p>Official schedule will be published after talks are selected.</p>
-    <p>Visit the <a href="https://osem.seagl.org/">CFP</a> to submit a talk!</p>
+    <p>The official SeaGL schedule is published on OSEM:
+    
+    <a href="https://osem.seagl.org/conference/seagl2015/schedule"></a>
```
Comment (ex-nerd) [inline schedule/2015.html]: It doesn't link to anything.  Maybe some variation of "click here to view it" ?

**Comment by tedmcox** [comment]
The page title has been fixed, too.

**Pull request merged**